index.js

```diff
@@ -6,38 +6,32 @@ exports.registerRoute = function (hook_name, args, cb) {
     var padName;
     
     createRandomPadName = function() {
-      var chars = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
-      var length = Math.floor((Math.random()*15)+8);
+      // the number of distinct chars (64) is chosen to ensure that
+      // the selection will be uniform when using the PRNG below
+      var chars = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz-_";
+      // the length of the pad name is chosen to get 120-bit security:
+      // log2(64^20) = 120
+      var string_length = 20;
+      // make room for 8-bit integer values that span from 0 to 255.
+      var randomarray = new Uint8Array(string_length);
+      // use Node.js PRNG to generate a "unique" sequence
+      const crypto = require('crypto');
+      crypto.randomFillSync(randomArray);
       var randomstring = '';
       for (var i = 0; i < length; i++)
```

Comment:
**string_**length?

Reply:
Fixed in 9aa09f2f002f9f3885b317220910fa4041a958be.